README.md

```diff
@@ -79,130 +87,17 @@ Add Alby to your browser
 
 ## 🚀 Quick Start
 
-Ensure you have
```

Comment:
Better move this back to the main readme:
![image](https://user-images.githubusercontent.com/1016218/204761466-a2fdd0aa-0325-4960-a762-beef4af45729.png)

Should be an easy quick start for devs.